Commiting AWS Batch Lab (#132)
aws-samples/aws-hpc-tutorials: content/05-AWS Batch/00-environment.md

tags = ["tutorial", "install", "AWS", "Batch"]
+++

This section modifies the Cloud9 environment that you created in the [Getting Started in the Cloud](/02-aws-getting-started.html) workshop in the following ways:

> - Expand the root volume to at least 20GB in capacity to allow container images to be built.

Reviewer: ```suggestion
- Expand the root volume to at least 20GiB in capacity to allow container images to be built.
```

- Upgrade to [AWS Command Line Interface (AWS CLI) Version 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html) in order to upload your container images to [Amazon Elastic Container Registry (ECR)](hhttps://aws.amazon.com/ecr/).

### Expand Cloud9 Root Volume
In this first step you will ensure the root volume of your Cloud9 instance has at least 20GB capacity in order for Docker images to be built locally.

1. Close any open Cloud9 browser sessions and follow this deep link to find [ your Cloud9 EC2 instance](https://console.aws.amazon.com/ec2/v2/home?#Instances:search=cloud9;sort=desc:launchTime).
2. Stop the instance if is running by selecting the instance and choosing **Instance state / Stop instance /**. ![Stop running instance](/images/aws-batch/root-volume-1.png)
3. Click on the Instance ID to see the instance details and click on the storage tab. Click on the Volume ID for the root volume. ![Click on the Storage tab](/images/aws-batch/root-volume-2.png)

> 4. Select the root volume for your Cloud9 instance and resize it to be at least 20GB in size by choosing **Actions / Modify volume /**. ![Modify the root volume](/images/aws-batch/root-volume-3.png)

Reviewer: ```suggestion
4. For actions, choose **Modify Volume** and for **Size** enter 20GiB. ![Modify the root volume](/images/aws-batch/root-volume-3.png)
```

> 5. Return to your [ Cloud9 EC2 instance](https://console.aws.amazon.com/ec2/v2/home?#Instances:search=cloud9;sort=desc:launchTime) and start it up by selecting the instance and choosing **Instance state / Start instance /**.

Reviewer: ```suggestion
5. Open Amazon EC2 Console [link](https://console.aws.amazon.com/ec2/v2/home?#Instances:search=cloud9;sort=desc:launchTime) to find your Cloud9 instance.
6. Select the Cloud9 instance
7. For **Instance state**, select **Reboot instance**
8. Open AWS Cloud9 Console [link](https://console.aws.amazon.com/cloud9) and select **Open IDE**
```

This section modifies the Cloud9 environment that you created in the [Getting Started in the Cloud](/02-aws-getting-started.html) workshop in the following ways:

> <<<<<<< HEAD

Reviewer: Please review those changes

- Expand the root volume to at least 20GiB in capacity to allow container images to be built.
=======
- Expand the root volume to at least 20GiB in capacity to allow container images to be built locally.
>>>>>>> 157dd70f87cf754d3a7043aa5352594adacfbbe2
- Upgrade to [AWS Command Line Interface (AWS CLI) Version 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html) in order to upload your container images to [Amazon Elastic Container Registry (ECR)](hhttps://aws.amazon.com/ecr/).

### Expand Cloud9 Root Volume
In this first step you will ensure the root volume of your Cloud9 instance has at least 20GiB capacity in order for Docker images to be built locally.

> <<<<<<< HEAD

Reviewer: Please review the changes
